cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -16,3 +20,39 @@ func bindFlagsToViper(cmd *cobra.Command, args []string) error {
 	})
 	return err
 }
+
+func setupClusterOptionFlags(cmd *cobra.Command, clusterOpt *clusterOptions) {
+	cmd.Flags().StringVarP(&clusterOpt.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
+	cmd.Flags().StringVar(&clusterOpt.bundlesOverride, "bundles-override", "", "Override default Bundles manifest (not recommended)")
+	cmd.Flags().StringVar(&clusterOpt.managementKubeconfig, "kubeconfig", "", "Management cluster kubeconfig file")
+}
+
```

Comment:
Would it be appropriate to move the constants (`Tinkerbell*` etc) here too given these funcs seem to be the primary consumers now?